### DISCUSSION TITLE
FD47D604-351B-5E70-8D62-E9474E7F13D0

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#191] [General]
# Solar System (JPL/Caltech Textures)

This addon modifies all planets and some moons in the Solar System with textures from Solar System Simulator, made by JPL/Caltech.
NOTE: These textures are very old and therefore the texture of Mercury as well as Saturn's moons is incomplete. Others textures are in black and white, such as the Galilean moons Europa, Ganymede and Callisto. Some are fictional, such as the texture of Jupiter, Saturn, Uranus, Neptune and Pluto.

https://celestia.mobi/addon?item=FD47D604-351B-5E70-8D62-E9474E7F13D0

**Comment by Zayedeisa2016** [comment]
I love solar system

**Comment by marinkabashi2019-cmyk** [comment]
I installed it and earth instantly became fully water i dont think it works for mobile
